lighthouse-core/lib/i18n/i18n-module.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  swapLocale: require('./swap-locale.js'),
+  ...require('./i18n.js'),
```

Comment:
We took a similar approach for our prototype:
- grabbing the file, bundling it with browserify and importing it. 

But hit a couple a blockers inside i18n.js that required some changes. @Christy Chen did a nice work cleaning the library and leaving it in a "minimal required" state, @paulirish and @connorjclark  what would be the best way for us to share this with the Lighthouse team (setting a PR with some changes, design document)?

Reply:
What were the blockers? can just start w/ sharing the branch, details on what the issue was (problems with bundling, or using it?)

Reply:
thanks, this was useful. i left a comment there